### PR TITLE
add: `gpu-screen-recorder`

### PR DESCRIPTION
**Pull request by bibelin**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [x] Contact devs (see below)
- [x] Add maintainer to pacscript

For build instructions and dependencies I looked at [gpu-screen-recorder-git AUR package](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=gpu-screen-recorder-git), ~~but for `source` I use a link to release tarball that can be found in [gpu-screen-recorder-gtk flatpak manifest](https://github.com/flathub/com.dec05eba.gpu_screen_recorder/blob/master/com.dec05eba.gpu_screen_recorder.yml#L41).~~

### CODE REVIEW

File: packages/gpu-screen-recorder/gpu-screen-recorder.pacscript [hunk 1]
```diff
@@ -0,0 +1,157 @@
+pkgname="gpu-screen-recorder"
+pkgdesc="A shadowplay-like screen recorder for Linux"
+pkgver="4.2.2"
+arch=("amd64")
+url="https://git.dec05eba.com/gpu-screen-recorder/about"
+repology="project:gpu-screen-recorder"
```
Comment (Elsie19) [inline packages/gpu-screen-recorder/gpu-screen-recorder.pacscript:6]:
```suggestion
repology=("project:gpu-screen-recorder")
```

File: packages/gpu-screen-recorder/gpu-screen-recorder.pacscript [hunk 2]
```diff
@@ -0,0 +1,157 @@
+pkgname="gpu-screen-recorder"
+pkgdesc="A shadowplay-like screen recorder for Linux"
+pkgver="4.2.2"
+arch=("amd64")
+url="https://git.dec05eba.com/gpu-screen-recorder/about"
```
Comment (Elsie19) [inline packages/gpu-screen-recorder/gpu-screen-recorder.pacscript:5]:
```suggestion
url='https://git.dec05eba.com/gpu-screen-recorder/about'
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Comment by bibelin** [comment]
@dec05eba, glad to see you here and thanks for the hints! :)

As now you are aware of the pacstall package, I have a few questions for you:
* Would you like to maintain the pacscript (similar to a PKGBUILD) yourself?
* Could you include a section in your documentation showing Pacstall as a valid method of installation for `gpu-screen-recorder` for Debian/Ubuntu users?
* Would you be interested in adding `gpu-screen-recorder-gtk` to pacstall repository as well? Tbh I don't use it myself, but I don't mind adding and maintaining a pacscript.

**Comment by dec05eba** [comment]
@bibelin 

>Would you like to maintain the pacscript (similar to a PKGBUILD) yourself?

>Would you be interested in adding gpu-screen-recorder-gtk to pacstall repository as well? Tbh I don't use it myself, but I don't mind adding and maintaining a pacscript.

I dont use debian/ubuntu so sorry no, I cant do that
>Could you include a section in your documentation showing Pacstall as a valid method of installation for gpu-screen-recorder for Debian/Ubuntu users?

Sure I'll do that when this is merged. There are some other distros that have packages as well so I'll add those as well there.

**Review by vigress8 — DISMISSED** [review]

File: packages/gpu-screen-recorder/gpu-screen-recorder.pacscript [hunk 3]
```diff
@@ -0,0 +1,157 @@
+pkgname="gpu-screen-recorder"
+pkgdesc="A shadowplay-like screen recorder for Linux"
+pkgver="4.2.2"
+arch=("amd64")
+url='https://git.dec05eba.com/gpu-screen-recorder/about'
+repology=("project:gpu-screen-recorder")
```
Comment (vigress8) [inline packages/gpu-screen-recorder/gpu-screen-recorder.pacscript:6]:
```suggestion
repology=("project: gpu-screen-recorder")
```
Tooling expects a space here

**Review by vigress8 — APPROVED** [review]

**Comment by bibelin** [comment]
@Elsie19 could you please review this again and merge if everything is okay?

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**